### PR TITLE
#4272 Mobile: Navigation bar doesn't disappear

### PR DESCRIPTION
**Pull request by VanilStorm**
**Related issue(s):**
* Fixes https://github.com/scandipwa/scandipwa/issues/4272

**Problem:**
* Mobile: Navigation bar disappears for few sec and then appears again when adding product to cart

**In this PR:**
* Created an additional scroll check

### CODE REVIEW

**Comment by aleksandrakorolova** [comment]
Still, can reproduce issue 
Notes: Navigation bar doesn't disappear when adding product to the cart to for the second time with condition: user didn't scroll page
https://drive.google.com/file/d/1YEvSzGUsk6_p_pYX0bfL5pjcfI6CDYtF/view?usp=sharing

**Comment by aleksandrakorolova** [comment]
Still, can reproduce with steps:
1. Open PDP
2. Open Write new review form
3. Fill form
4. Submit
https://drive.google.com/file/d/1Ys_Lau9H2h-hOc5ZsWyLMH-ZIjcMfQaP/view?usp=sharing

File: packages/scandipwa/src/component/NavigationTabs/NavigationTabs.container.js [hunk 1]
```diff
@@ -56,10 +58,18 @@ export const DEFAULT_NAVIGATION_TABS_STATE = { name: MENU_TAB };
 
 /** @namespace Component/NavigationTabs/Container */
 export class NavigationTabsContainer extends NavigationAbstractContainer {
+    static propTypes = {
+        isNotificationVisable: PropTypes.bool.isRequired
```
Comment (zans-laksa) [inline packages/scandipwa/src/component/NavigationTabs/NavigationTabs.container.js:62]: Please fix the typo.
Visible, not visable.

File: packages/scandipwa/src/component/NavigationTabs/NavigationTabs.container.js [hunk 2]
```diff
@@ -136,6 +146,9 @@ export class NavigationTabsContainer extends NavigationAbstractContainer {
         const offset = window.innerHeight + window.pageYOffset;
         const height = doc.scrollHeight;
 
+        const { isNotificationVisable } = this.props;
+        const { isScrolling } = this.state;
```
Comment (zans-laksa) [inline packages/scandipwa/src/component/NavigationTabs/NavigationTabs.container.js:150]: I believe the `isScrolling` is a misnomer here.
You don't check whether the page is scrolling here.
You just set the boolean flag here and exit the function if the flag is true.
Please put the proper name.

**Review by zans-laksa — CHANGES_REQUESTED** [review]

**Comment by aleksandrakorolova** [comment]
Closed because issue was fixed in #4579

**Pull request closed without merging**